package.json

```diff
@@ -4,11 +4,13 @@
   "repository": "https://github.com/dimagi/commcare-hq",
   "description": "CommcareHQ Javascript tests",
   "dependencies": {
+    "biginteger": "^1.0.3",
```

Comment:
try it out with this branch: https://github.com/dimagi/js-xpath/tree/encapsulate-number-libs